src/main/resources/application.yaml

```diff
@@ -28,7 +28,7 @@ spring:
     name: Translation Service
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://${TS_TRANSLATION_SERVICE_DB_HOST:localhost}:${TS_TRANSLATION_SERVICE_DB_PORT:6432}/${TS_TRANSLATION_SERVICE_DB_NAME:ts_translation_service}${TS_TRANSLATION_SERVICE_DB_OPTIONS:}
+    url: jdbc:postgresql://${TS_TRANSLATION_SERVICE_DB_HOST:localhost}:${TS_TRANSLATION_SERVICE_DB_PORT:6969}/${TS_TRANSLATION_SERVICE_DB_NAME:ts_translation_service}${TS_TRANSLATION_SERVICE_DB_OPTIONS:}
```

Comment:
This should match the port specified in Camerons WLTS-13 PR (https://github.com/hmcts/ccd-docker/pull/176/files)

Reply:
6969 is the port defined in docker-compose.yml  for ts-translation-service.  If there now a defined port in ccd-docker cool .  We need to change master but not in this PR as it is not part of the implementation .